torch/onnx/utils.py

```diff
@@ -1798,18 +1797,19 @@ def _run_symbolic_function(
             domain = namespace
         symbolic_function_name = f"{domain}::{op_name}"
 
-        symbolic_function_group = registration.registry.get_function_group(
-            symbolic_function_name
-        )
-        if symbolic_function_group is not None:
```

Comment:
Did you change your mind on `is not None` 🤔

Reply:
Since we are hiding the check inside `dispatch.symbolics`, we can do a single check (or try catch).

symbolic_function_group is explicitly needed before so it didn't make sense to call `is_registered_op` first, where the None check would still be needed for mypy